boards/arc/em_starterkit/doc/board.rst

```diff
@@ -174,122 +181,55 @@ Parity:    None
 Stopbits:  1
 ========= =====
 
-Debugging
-==========
-Before you can debug, you will need to download and install the
-Synopsys versions of ARC GNU tools. Unfortunately the Zephyr-SDK versions
-of openocd and gdb have some functionality limitations and don't yet
-work well with the ARC EM Starter Kit.
-
-The Synopsys tools are found on GITHUB here:
-
-https://github.com/foss-for-synopsys-dwc-arc-processors/toolchain/releases/tag/arc-2016.03
+Using the latest version of Zephyr SDK(>=0.9), you can debug and flash
+EM Starterkit directly.
 
-For Linux, you will need:
+Boot up the SOC by pressing the "C" button. Be sure the digilent cable
```

Comment:
to be clear,is this a "C" button on the board?  Also, if the cable needs to be connected before pressing the "C" button, put that step first rather than as an afterthought.